.github/ISSUE_TEMPLATE/divulgacao.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Divulgação do evento Python Floripa Online
+about: Modelo de mensagem de divulgação do evento nas redes sociais da comunidade
+title: "Modelo de mensagem de divulgação do evento nas redes sociais da comunidade"
+labels: ''
+assignees: ''
+
+---
+
+# Descrição
+
+Modelo de mensagem usada para divulgação dos eventos da Python Floripa Online nas [redes sociais da comunidade](https://floripa.sc.python.org.br/contato/). Este modelo deve ser preenchido nos locais com os símbolos **<>**, com os itens indicados.
+
+Este modelo é apenas um lembrete para quais informações devem ser passadas. Libere a imaginação, e coloque uma piada nerd no meio das mensagens! :video_game: :floppy_disk: :game_die:
+
+_**Saudações pythonistas, passando para avisar que <DATA> teremos mais uma edição de Python Floripa Online :) <LINK EVENTO MEETUP> <LINK EVENTO YOUTUBE> Não quer perder nenhuma novidade da comunidade Python de Florianópolis? Acompanhe todas nossas redes sociais https://floripa.sc.python.org.br/contato/**_
```

Comment:
Fiquei na dúvida sobre esse template. Se for só para guardar o texto, talvez melhor guardar na wiki ou até mesmo na issue do evento, uma forma legal seria usar a tag `<details>` ([doc](https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab)).

<details>
<summary>Template de mensagem para redes sociais</summary>

_**Saudações pythonistas, passando para avisar que <DATA> teremos mais uma edição de Python Floripa Online :) <LINK EVENTO MEETUP> <LINK EVENTO YOUTUBE> Não quer perder nenhuma novidade da comunidade Python de Florianópolis? Acompanhe todas nossas redes sociais https://floripa.sc.python.org.br/contato/**_
</details>

Reply:
Concordo, pode ficar confuso como outro template. Vou adicionar como um collapse (`<details>`) na issue do evento